### PR TITLE
update setup scripts

### PR DESCRIPTION
**Pull request by TomWoodward**
@m1yag1 please make the makefile less gross

please @m1yag1 you are our only hope

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -23,11 +23,27 @@ clean:
 	rm -f node_modules/.uptodate .pydeps
 	rm -rf build
 
+.PHONY: env
+env:
```
Comment (m1yag1) [inline Makefile:27]: I kinda feel like a comment summarizing what is happening here would be useful for those not ninjas in stream editing.

File: Makefile [hunk 2]
```diff
@@ -23,11 +23,27 @@ clean:
...
 ## Run the development H server locally
 .PHONY: dev
-dev: build/manifest.json .pydeps
-	@bin/hypothesis --dev init
-	@bin/hypothesis devserver
+dev:
+	@echo "initializing hypothesis..."
+	@bin/hypothesis init
+
+ifneq ($(add_credentials_to),)
```
Comment (m1yag1) [inline Makefile:36]: I cant say i've seen `add_credentials_to` before. How is this working?

Comment (TomWoodward) [reply Makefile:36]: i'm expecting that to be passed in via command line ala https://github.com/openstax/tutor-server/pull/1756/files#diff-4e5e90c6228fd48698d074241c2ba760R32

i could for sure update the help section for both of these commands

**Review by m1yag1 — APPROVED** [review]
Adding context in the help section would be good. other than that this looks good to go.

**Pull request merged**